scripts/ProcessTemplates.ps1

```diff
@@ -256,6 +256,14 @@ Function CopyFromEnuIfNotExist() {
         $fileName = $enuFile.Name
         $destinationFile = "$fullName\$fileName"
         if (!(Test-Path $destinationFile)) {
+            if ($fileName -like "*.workbook") {

```

Comment:
comments would be good.  if I interpret this right, it is skipping files from the *localized* repo if the same file name doesn't exist in the English one?

so in this case, should the output show `>>>>> skipping...` and then just below it show `[#Warning: missing file]…` ?

Reply:
in the build I see this:
[INFO] [133] >>>>>>>>>>>>>>>>>> Skipping .workbook in S:\scripts\Application-Insights-Workbooks.cs-cz\Workbooks\View Designer\View Designer Transition Guide\Transition Guide.workbook <<<<<<<<

[INFO] [134] >>>>>>>>>>>>>>>>>> Skipping .workbook in S:\scripts\Application-Insights-Workbooks.cs-cz\Workbooks\View Designer\View Designer Transition Guide\Transition Guide.workbook <<<<<<<<
```
[INFO] [135] [#WARNING: missing File]: copying file S:\Workbooks\Virtual Machine Scale Sets - Network Dependencies\categoryResources.json to S:\scripts\Application-Insights-Workbooks.cs-cz\Workbooks\Virtual Machine Scale Sets - Network Dependencies

[INFO] [136] [#WARNING: missing File]: ...found S:\scripts\Application-Insights-Workbooks.cs-cz\Workbooks\Virtual Machine Scale Sets - Network Dependencies\categoryResources.json and replacing "en-us" with "cs-cz"
``` 

which implies to me that it skipped the file completely, does that mean that the English one won't be there in this language either?  or does some other process handle that?

Reply:
i will add more comments.

for the file that's missing in Loc compared with Enu, this checks that .workbook is the missing file and then get all .workbook files in the Loc folder to see if there's already a .workbook in that folder and skip processing. current folder ($fullname) is the Loc branch it's processing

Reply:
skipping means it detected another .workbook in the Loc branch so it won't copy over the enu version; this means the Loc branch will use an older .workbook version until the next time loc process sync the files from master. missing file indicates it detected missing files in Loc branch compared with enu and it's copying over that file before merging into a final json file